src/components/afm/BulletChart.tsx

```diff
@@ -0,0 +1,19 @@
+// (C) 2007-2020 GoodData Corporation
+import * as React from "react";
```

Comment:
Check with Lupko if we want this AFM component. AFM should be already deprecated and you can skip this level and go directly from bucket interface into dataSources as I think PivotTable do

Reply:
They did the same for Xirr so we expect it's OK to continue like that. I'll ask  Lupko or Dan

Reply:
I asked Dan and he's OK with that